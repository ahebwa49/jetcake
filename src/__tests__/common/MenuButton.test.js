import React from "react";
import configureStore from "redux-mock-store";
import { shallow, mount } from "enzyme";
import { Provider } from "react-redux";
import toJson from "enzyme-to-json";

import MenuButton from "../../common/MenuButton";

const mockStore = configureStore([]);

let component;

component = shallow(<MenuButton />);

test("MenuButton component renders without crashing", () => {
  expect(toJson(component)).toMatchSnapshot();
});
