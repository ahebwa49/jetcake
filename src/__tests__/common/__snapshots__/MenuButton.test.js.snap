// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuButton component renders without crashing 1`] = `
<div>
  <div
    style={
      Object {
        "cursor": "pointer",
        "display": "relative",
        "top": "10px",
      }
    }
  >
    <div
      style={
        Object {
          "backgroundColor": "#333",
          "height": "3px",
          "margin": "3px 0px",
          "marginBottom": "3px",
          "width": "20px",
        }
      }
    />
    <div
      style={
        Object {
          "backgroundColor": "#333",
          "height": "3px",
          "margin": "3px 0px",
          "width": "20px",
        }
      }
    />
    <div
      style={
        Object {
          "backgroundColor": "#333",
          "height": "3px",
          "margin": "3px 0px",
          "marginTop": "3px",
          "width": "20px",
        }
      }
    />
  </div>
</div>
`;
